index.js

```diff
@@ -12,6 +12,23 @@ module.exports = (robot, _, Settings = require('./lib/settings')) => {
 
     if (defaultBranch && settingsModified) {
       return Settings.sync(context.github, context.repo())
+    } else if (!defaultBranch && settingsModified) {
+      let settingChangeCommits = payload.commits.filter(commit => { // get all commits that change settings.yml
+        return commit.added.includes(Settings.FILE_NAME) ||
+        commit.modified.includes(Settings.FILE_NAME)
+      })
+      settingChangeCommits.forEach(async function (commit, index, array) { // create status for each commit that changed settings.yml
+        try {
+          await context.github.repos.createStatus({
+            owner: payload.repository.owner.login,
+            repo: payload.repository.name,
```

Comment:
You can use `context.repo` for shorthand to extract `owner` and `repo`. See https://probot.github.io/api/latest/Context.html#repo